Dockerfile

```
@@ -0,0 +1,9 @@
+FROM quay.io/giantswarm/alpine:3.19.0
```

You can also use gsoci, which is likely more future proof.

```suggestion
FROM gsoci.azurecr.io/giantswarm/alpine:3.19.0
```